lessons/13-css-external-files/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css" />
+    <style>
+        .nav-one {
+            color: #00aced;
+        }
```

Comment:
this is repeated in each file, you could likely create a class in your external `.css` file and reuse it as the active class 😄